src/client/css/partials/error.css

```diff
@@ -0,0 +1,24 @@
+.error-page {
+  background-image: url('/images/error-bg.svg');
+  background-position: center;
+  background-repeat: no-repeat;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  padding: var(--padding-xl);
+}
+
+.error-page h1 {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  gap: var(--padding-sm);
+}
+
+.error-page .error-code {
+  font: var(--h3-font);
+  font-weight: bold;
+  font-size: 10rem;
+  color: var(--purple-60);
+  text-shadow: 0.75rem 0 0 var(--purple-20);
+}
```

Comment:
Empty line missing.

Reply:
Here too, somehow it got automatically removed, and now it no longer does it... Fixed in 8199089a, but I've got to figure out what's going wrong here sometime :/